pkg/engine/vmextensions.go

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package engine
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/Azure/aks-engine/pkg/api"
+	"github.com/Azure/aks-engine/pkg/api/common"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-10-01/compute"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+//func createManagedIdentityExtension(cs api.ContainerService) VirtualMachineExtensionARM {
```

Comment:
Is this // TODO work for functional equivalency w/ current master? Or experimental stuff? If the latter let's get rid of. If the former let's specify w/ a // TODO comment exactly what functional gap this commented out code solves.

Reply:
This is defunct basically. Will remove.